src/Pim/index.tsx

```diff
@@ -115,10 +115,10 @@ type CollectionRoutesPropsType = RouteComponentProps<{}> & {
 
 const styles = (theme: any) => ({
   button: {
-    marginLeft: theme.spacing.unit,
+    marginLeft: theme.spacing(1),
```

Comment:
This is a fine change, as unit is deprecated for all I remember, but this change is unrelated to your PR. Please don't mix unrelated changes. I'd be happy to merge this if you open a different PR with this fix.